dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/event/data/JdbcEnrollmentAnalyticsManager.java

```diff
@@ -140,8 +140,19 @@ private void getEnrollments( EventQueryParams params, Grid grid, String sql )
 
         SqlRowSet rowSet = jdbcTemplate.queryForRowSet( sql );
 
+        int rowsRed = 0;
```

Comment:
Do we need the same rules for events as well? Or this issue only reproduces in enrollments?